app/main.py

```
@@ -1 +1,20 @@
 # write your code here
+def copy_file(command: str) -> None:
+    try:
+        cp = command.split(" ")[0]
+        if cp != "cp":
+            return
+        src = command.split(" ")[1]
+        dest = command.split(" ")[2]
+    except IndexError:
+        pass
```

The `pass` statement in the `except` block silently ignores the `IndexError`. It would be better to handle this exception, for example, by printing an error message to inform the user about the incorrect command format.